src/dialect.rs

```diff
@@ -6,8 +6,8 @@ use crate::reduction::Response;
 #[derive(Clone, Copy, Eq, PartialEq)]
 pub enum OperatorSet {
     Default,
-    BLS,
     Keccak, // keccak256 operator
+    Unknown,
```

Comment:
I think it would make sense to, briefly, describe the issue we just introduced and how `Default` and `Unknown` differ and solve the problem.